io/iouring-wrapper.cpp

```diff
@@ -664,18 +698,17 @@ int iouring_unregister_files(int fd) {
     return ee->register_unregister_files(fd, false);
 }
 
-__attribute__((noinline))
-static iouringEngine* new_iouring(bool is_master) {
-    LOG_INFO("Init event engine: iouring ", VALUE(is_master));
-    return NewObj<iouringEngine>(is_master) -> init();
+void* new_iouring_event_engine(iouring_args args) {
+    LOG_INFO("Init event engine: iouring ",
+        make_named_value("is_master",     args.is_master),
+        make_named_value("setup_sqpoll",  args.setup_sqpoll),
+        make_named_value("setup_sq_aff",  args.setup_sq_aff),
+        make_named_value("sq_thread_cpu", args.sq_thread_cpu));
+    auto uring = NewObj<iouringEngine>() -> init(args);
+    if (args.is_master) return uring;
```

Comment:
uring's type is MasterEventEngine? Don't need to do conversion?

Reply:
no need for the 1st parent class